runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/environment/ProcessManager.java

```diff
@@ -234,9 +231,7 @@ public void run() {
               // Graceful shutdown period
               Thread.sleep(200);
               break;
-            } catch (InterruptedException e) {
-              Thread.currentThread().interrupt();
-              throw new RuntimeException(e);
+            } catch (InterruptedException ignored) {
```

Comment:
Add comment on why the exception is ignored.

Reply:
Will do. Ignoring it because there is not point to throw it here, we want to shutdown the processes after the grace period expired here.